src/explore-education-statistics-common/src/modules/table-tool/utils/__tests__/generateTableTitle.test.ts

```diff
@@ -81,20 +81,30 @@ describe('generateTableTitle', () => {
       },
     });
 
-    expect(result).toBe(
+    expect(title).toBe(
+      "Authorised absence rate for 'Absence by characteristic' in England for 2015/16",
+    );
+  });
+
+  test('with no filters', () => {
+    const title = generateTableTitle({
+      ...testFullTableMeta,
```

Comment:
Could rename this to `testMeta` as you did in another test file in this PR, but I'm not overly fussed :)

Reply:
Done!